scripts/tests/twister/test_harness.py

```diff
@@ -34,6 +43,348 @@ def process_logs(harness, logs):
         harness.handle(line)
 
 
+TEST_DATA_11 = [('RunID: 12345', False, False, False, None, True),
+                ('PROJECT EXECUTION SUCCESSFUL', False, False, False, 'passed', False),
+                ('PROJECT EXECUTION SUCCESSFUL', True, False, False, 'failed', False),
+                ('PROJECT EXECUTION FAILED', False, False, False, 'failed', False),
+                ('ZEPHYR FATAL ERROR', False, True, False, None, False),
+                ('GCOV_COVERAGE_DUMP_START', None, None, True, None, False),
+                ('GCOV_COVERAGE_DUMP_END', None, None, False, None, False),]
+@pytest.mark.parametrize(
+    "line, fault, fail_on_fault, cap_cov, exp_stat, exp_id",
+    TEST_DATA_11,
```

Comment:
We have `TEST_DATA_2`, `TEST_DATA_3`, `TEST_DATA_4`, `TEST_DATA_11`, `TEST_DATA_12`, `TEST_DATA_13` and `TEST_DATA_14` in this file. Those could be reorganised to 1-7.